src/http/ngx_http_upstream.h

```diff
@@ -24,20 +24,22 @@
 #define NGX_HTTP_UPSTREAM_FT_HTTP_502        0x00000020
 #define NGX_HTTP_UPSTREAM_FT_HTTP_503        0x00000040
 #define NGX_HTTP_UPSTREAM_FT_HTTP_504        0x00000080
-#define NGX_HTTP_UPSTREAM_FT_HTTP_403        0x00000100
-#define NGX_HTTP_UPSTREAM_FT_HTTP_404        0x00000200
-#define NGX_HTTP_UPSTREAM_FT_HTTP_429        0x00000400
-#define NGX_HTTP_UPSTREAM_FT_UPDATING        0x00000800
-#define NGX_HTTP_UPSTREAM_FT_BUSY_LOCK       0x00001000
-#define NGX_HTTP_UPSTREAM_FT_MAX_WAITING     0x00002000
-#define NGX_HTTP_UPSTREAM_FT_NON_IDEMPOTENT  0x00004000
+#define NGX_HTTP_UPSTREAM_FT_HTTP_401        0x00000100
+#define NGX_HTTP_UPSTREAM_FT_HTTP_403        0x00000200
```

Comment:
Please tell me if it's OK to change these values though, I have not references to these on nginx/nginx, but also I'm not sure if 3rd party nginx modules depend on this or not.

There is https://github.com/nginx/nginx/commit/1a983a0 which has changed these values.